gcloud-java-dns/src/main/java/com/google/gcloud/dns/testing/package-info.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A testing helper for Google Cloud DNS.
+ *
+ * <p>A simple usage example:
+ * Before the test:
+ * <pre> {@code
+ * long delay = 0; // minimum delay in processing change requests (in ms)
```

Comment:
Maybe "for processing"? Also do you think its worth mentioning 0 means synchronous?